lib/Db/MessageMapper.php

```diff
@@ -116,6 +119,12 @@ public function insertBulk(Message ...$messages): void {
 		foreach ($messages as $message) {
 			$qb1->setParameter('uid', $message->getUid(), IQueryBuilder::PARAM_INT);
 			$qb1->setParameter('message_id', $message->getMessageId(), IQueryBuilder::PARAM_STR);
+			$inReplyTo = $message->getInReplyTo();
+			$qb1->setParameter('in_reply_to', $inReplyTo, $inReplyTo === null ? IQueryBuilder::PARAM_NULL : IQueryBuilder::PARAM_STR);
```

Comment:
can you check this before and use `$qb->expr()->isNull()` instead?

Reply:
I don't get it. Change what?

Reply:
Ignore, this is on insert... not comparing